### ISSUE TITLE
YEI 3space sensor can get into reset loop

### ISSUE THREAD

**Issue by rpavlik** [vrpn/vrpn#72]
In a server where something might happen that takes a while in the mainloop (in my case, OSVR server running a hardware autodetect to pick up the Hydra, which has a slightly lengthy startup process), the 3Space sensor gets into a loop: 

```
VRPN Error
 (0) from org_opengoggles_bundled_Multiserver/YEI_3Space_Sensor0: vrpn_YEI_3Space::handle_report(): Invalid confidence, resetting
```

is repeated endlessly. If I kill the server and start it again, it's OK again.

**Comment by russell-taylor** [comment]
Reproduced by adding a Sleep() into the main loop of vrpn.C in the server code.

**Comment by russell-taylor** [comment]
Fixed in commit bf48e068fa7fa02b48055bf4bfc666fbc59c1509
